packages/editor/types.ts

```
@@ -81,12 +81,37 @@ export enum EditorCommandType {
+	sourceJs: string;
+	uri?: undefined;
+};
+type ContentScriptJs = ContentScriptUriSource|ContentScriptInlineSource;
+
 // Because the editor package can run in a WebView, plugin content scripts
 // need to be provided as text, rather than as file paths.
 export interface ContentScriptData {
 	pluginId: string;
 	contentScriptId: string;
-	contentScriptJs: ()=> Promise<string>;
+	contentScriptJs: (context: ContentScriptLoadOptions)=> Promise<ContentScriptJs>;
```

ESLint

[error] 'context' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)